### PR TITLE
Upgrade dependency version

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -30,16 +30,16 @@ rapis_version=22.12.0
 commons_cli_version=1.6.0
 commons_compress_version=1.26.1
 commons_csv_version=1.10.0
-commons_logging_version=1.3.0
+commons_logging_version=1.3.1
 gson_version=2.10.1
-jna_version=5.13.0
-slf4j_version=1.7.36
```
Comment (frankfliu) [inline gradle.properties:36]: Keep slf4j 1.x

File: gradle.properties [hunk 2]
```diff
@@ -30,16 +30,16 @@ rapis_version=22.12.0
...
+slf4j_version=2.0.12
 log4j_slf4j_version=2.23.1
-awssdk_version=2.25.13
-hadoop_version=3.3.6
-javacpp_version=1.5.9
-javacv_version=1.5.9
-ffmpeg_version=6.0-1.5.9
+awssdk_version=2.25.17
+hadoop_version=3.4.0
+javacpp_version=1.5.10
```
Comment (frankfliu) [inline gradle.properties:40]: Keep java cpp 1.5.9, we can upgrade in next release

File: gradle.properties [hunk 3]
```diff
@@ -30,16 +30,16 @@ rapis_version=22.12.0
...
-commons_logging_version=1.3.0
+commons_logging_version=1.3.1
 gson_version=2.10.1
-jna_version=5.13.0
-slf4j_version=1.7.36
+jna_version=5.14.0
+slf4j_version=2.0.12
 log4j_slf4j_version=2.23.1
-awssdk_version=2.25.13
-hadoop_version=3.3.6
```
Comment (frankfliu) [inline gradle.properties:39]: Let upgrade hadoop in next release

**Review by frankfliu — COMMENTED** [review]
Please

File: gradle.properties (hunk 1 above)
Comment (xyang16) [reply gradle.properties:36]: Done

File: gradle.properties (hunk 3 above)
Comment (xyang16) [reply gradle.properties:39]: Done

File: gradle.properties (hunk 2 above)
Comment (xyang16) [reply gradle.properties]: Done

File: gradle.properties [hunk 4]
```diff
@@ -30,16 +30,16 @@ rapis_version=22.12.0
...
 slf4j_version=1.7.36
 log4j_slf4j_version=2.23.1
-awssdk_version=2.25.13
+awssdk_version=2.25.17
 hadoop_version=3.3.6
 javacpp_version=1.5.9
-javacv_version=1.5.9
-ffmpeg_version=6.0-1.5.9
+javacv_version=1.5.10
+ffmpeg_version=6.1.1-1.5.10
```
Comment (frankfliu) [inline gradle.properties:42]: They are using javacpp 1.5.10 as well, no upgrade for now.

Comment (xyang16) [reply gradle.properties]: Done

**Review by frankfliu — APPROVED** [review]

**Pull request merged**